pkg/scheduler/cache/cache.go

```diff
@@ -519,27 +527,31 @@ func newSchedulerCache(config *rest.Config, schedulerNames []string, defaultQueu
 		nodeQueue:           workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter()),
 		DeletedJobs:         workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter()),
 		kubeClient:          kubeClient,
+		dynClient:           dynamic.NewForConfigOrDie(config),
```

Comment:
also need init fake client in `newMockSchedulerCache`.

Reply:
same as above.